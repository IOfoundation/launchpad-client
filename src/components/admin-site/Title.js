import React from 'react';
import {withStyles} from '@material-ui/core/styles';
import {PropTypes} from 'prop-types';
import Grid from '@material-ui/core/Grid';

import Buttons from './Buttons';

const Title = props => {
  const {
    cancelClicked,
    classes,
    disableSubmit,
    extraClicked,
    extraLabel,
    hideCancelAction,
    noMargin,
    submitClicked,
    submitLabel,
    subTitle,
    titleText,
  } = props;
  const containerClasses = [classes.Container];

  if (noMargin) {
    containerClasses.push('margin-0');
  }

  return (
    <Grid
      className={containerClasses.join(' ')}
      container={true}
      justify="space-between"
    >
      <Grid item={true} xs={12} md={5}>
        <h1 className={classes.Title}>{titleText}</h1>
        {subTitle && <h2 className={classes.subTitle}>{subTitle}</h2>}
      </Grid>
      <Grid item={true} xs={12} md={7}>
        <Buttons
          cancelClicked={cancelClicked}
          disableSubmit={disableSubmit}
          extraClicked={extraClicked}
          extraLabel={extraLabel}
          hideCancelAction={hideCancelAction}
          submitClicked={submitClicked}
          submitLabel={submitLabel}
        />
      </Grid>
    </Grid>
  );
};

const styles = theme => {
  return {
    Buttons: {
      flex: 1,
      display: 'flex',
      alignItems: 'center',
      justifyContent: 'flex-end',
      paddingBottom: '28px',
      '& .btn': {
        width: 'auto',
        padding: '9px 25px',
        fontSize: '14px',
        fontFamily: '"proxima-nova-semi", Georgia, sans-serif',
        margin: 0,
      },
      '& a': {
        color: '#7B7C7E',
      },
      [theme.breakpoints.down('sm')]: {
        justifyContent: 'flex-start',
      },
    },
    Container: {
      borderBottom: '2px solid black',
      marginBottom: '28px',
    },
    Title: {
      fontSize: '34px',
      lineHeight: '40px',
      fontFamily: '"proxima-nova-bold", Georgia, sans-serif',
      color: '#070709',
      [theme.breakpoints.down('sm')]: {
        marginBottom: '16px',
      },
    },
    subTitle: {
      fontSize: '14px',
      color: '#070709',
    },
  };
};

Title.propTypes = {
  cancelClicked: PropTypes.func,
  classes: PropTypes.shape({
    Buttons: PropTypes.string,
    Container: PropTypes.string,
    Title: PropTypes.string,
    subTitle: PropTypes.string,
  }),
  disableSubmit: PropTypes.bool,
  extraClicked: PropTypes.func,
  extraLabel: PropTypes.string,
  hideCancelAction: PropTypes.bool,
  noMargin: PropTypes.bool,
  submitClicked: PropTypes.func,
  submitLabel: PropTypes.string,
  subTitle: PropTypes.string,
  titleText: PropTypes.string,
};

export default withStyles(styles)(Title);
