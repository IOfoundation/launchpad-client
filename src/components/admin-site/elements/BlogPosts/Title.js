import React from 'react';
import {PropTypes} from 'prop-types';
import Grid from '@material-ui/core/Grid';

import TitleField from '@Shared/FormElements/TitleField';

export const Title = props => {
  const {errors, touched, handleBlur, handleChange, values} = props;

  return (
    <div className="m-bot-8">
      <Grid container={true}>
        <Grid item={true} xs={12}>
          <TitleField
            autocomplete="off"
            error={touched.title && Boolean(errors.title)}
            errors={errors}
            handleBlur={handleBlur}
            handleChange={handleChange}
            id="title"
            name="title"
            label="Add a title..."
            value={values.title}
            multiline={true}
          />
        </Grid>
      </Grid>
    </div>
  );
};

Title.propTypes = {
  classes: PropTypes.shape({}),
  errors: PropTypes.shape({}),
  handleBlur: PropTypes.func,
  handleChange: PropTypes.func,
  touched: PropTypes.shape({}),
  values: PropTypes.shape({}),
};

export default Title;
