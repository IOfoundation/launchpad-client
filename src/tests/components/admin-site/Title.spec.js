import React from 'react';
import {mount} from 'enzyme';
import toJson from 'enzyme-to-json';

import Title from '../../../components/admin-site/Title';

const props = {
  submitClicked: jest.fn(),
  submitLabel: 'label',
};

describe('Admin-site <Title />', () => {
  it('renders snapshot of Title', () => {
    const wrapper = mount(<Title {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
