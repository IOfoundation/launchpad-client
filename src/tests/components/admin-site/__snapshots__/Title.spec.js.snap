// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Admin-site <Title /> renders snapshot of Title 1`] = `
<WithStyles(Title)>
  <Title
    classes={
      Object {
        "Buttons": "Title-Buttons-1",
        "Container": "Title-Container-2",
        "Title": "Title-Title-3",
      }
    }
  >
    <WithStyles(Grid)
      className="Title-Container-2"
      container={true}
      justify="space-between"
    >
      <Grid
        alignContent="stretch"
        alignItems="stretch"
        className="Title-Container-2"
        classes={
          Object {
            "align-content-xs-center": "MuiGrid-align-content-xs-center-16",
            "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-18",
            "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-17",
            "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-20",
            "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-19",
            "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-15",
            "align-items-xs-center": "MuiGrid-align-items-xs-center-12",
            "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-14",
            "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-13",
            "container": "MuiGrid-container-4",
            "direction-xs-column": "MuiGrid-direction-xs-column-7",
            "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-8",
            "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-9",
            "grid-lg-1": "MuiGrid-grid-lg-1-75",
            "grid-lg-10": "MuiGrid-grid-lg-10-84",
            "grid-lg-11": "MuiGrid-grid-lg-11-85",
            "grid-lg-12": "MuiGrid-grid-lg-12-86",
            "grid-lg-2": "MuiGrid-grid-lg-2-76",
            "grid-lg-3": "MuiGrid-grid-lg-3-77",
            "grid-lg-4": "MuiGrid-grid-lg-4-78",
            "grid-lg-5": "MuiGrid-grid-lg-5-79",
            "grid-lg-6": "MuiGrid-grid-lg-6-80",
            "grid-lg-7": "MuiGrid-grid-lg-7-81",
            "grid-lg-8": "MuiGrid-grid-lg-8-82",
            "grid-lg-9": "MuiGrid-grid-lg-9-83",
            "grid-lg-auto": "MuiGrid-grid-lg-auto-73",
            "grid-lg-true": "MuiGrid-grid-lg-true-74",
            "grid-md-1": "MuiGrid-grid-md-1-61",
            "grid-md-10": "MuiGrid-grid-md-10-70",
            "grid-md-11": "MuiGrid-grid-md-11-71",
            "grid-md-12": "MuiGrid-grid-md-12-72",
            "grid-md-2": "MuiGrid-grid-md-2-62",
            "grid-md-3": "MuiGrid-grid-md-3-63",
            "grid-md-4": "MuiGrid-grid-md-4-64",
            "grid-md-5": "MuiGrid-grid-md-5-65",
            "grid-md-6": "MuiGrid-grid-md-6-66",
            "grid-md-7": "MuiGrid-grid-md-7-67",
            "grid-md-8": "MuiGrid-grid-md-8-68",
            "grid-md-9": "MuiGrid-grid-md-9-69",
            "grid-md-auto": "MuiGrid-grid-md-auto-59",
            "grid-md-true": "MuiGrid-grid-md-true-60",
            "grid-sm-1": "MuiGrid-grid-sm-1-47",
            "grid-sm-10": "MuiGrid-grid-sm-10-56",
            "grid-sm-11": "MuiGrid-grid-sm-11-57",
            "grid-sm-12": "MuiGrid-grid-sm-12-58",
            "grid-sm-2": "MuiGrid-grid-sm-2-48",
            "grid-sm-3": "MuiGrid-grid-sm-3-49",
            "grid-sm-4": "MuiGrid-grid-sm-4-50",
            "grid-sm-5": "MuiGrid-grid-sm-5-51",
            "grid-sm-6": "MuiGrid-grid-sm-6-52",
            "grid-sm-7": "MuiGrid-grid-sm-7-53",
            "grid-sm-8": "MuiGrid-grid-sm-8-54",
            "grid-sm-9": "MuiGrid-grid-sm-9-55",
            "grid-sm-auto": "MuiGrid-grid-sm-auto-45",
            "grid-sm-true": "MuiGrid-grid-sm-true-46",
            "grid-xl-1": "MuiGrid-grid-xl-1-89",
            "grid-xl-10": "MuiGrid-grid-xl-10-98",
            "grid-xl-11": "MuiGrid-grid-xl-11-99",
            "grid-xl-12": "MuiGrid-grid-xl-12-100",
            "grid-xl-2": "MuiGrid-grid-xl-2-90",
            "grid-xl-3": "MuiGrid-grid-xl-3-91",
            "grid-xl-4": "MuiGrid-grid-xl-4-92",
            "grid-xl-5": "MuiGrid-grid-xl-5-93",
            "grid-xl-6": "MuiGrid-grid-xl-6-94",
            "grid-xl-7": "MuiGrid-grid-xl-7-95",
            "grid-xl-8": "MuiGrid-grid-xl-8-96",
            "grid-xl-9": "MuiGrid-grid-xl-9-97",
            "grid-xl-auto": "MuiGrid-grid-xl-auto-87",
            "grid-xl-true": "MuiGrid-grid-xl-true-88",
            "grid-xs-1": "MuiGrid-grid-xs-1-33",
            "grid-xs-10": "MuiGrid-grid-xs-10-42",
            "grid-xs-11": "MuiGrid-grid-xs-11-43",
            "grid-xs-12": "MuiGrid-grid-xs-12-44",
            "grid-xs-2": "MuiGrid-grid-xs-2-34",
            "grid-xs-3": "MuiGrid-grid-xs-3-35",
            "grid-xs-4": "MuiGrid-grid-xs-4-36",
            "grid-xs-5": "MuiGrid-grid-xs-5-37",
            "grid-xs-6": "MuiGrid-grid-xs-6-38",
            "grid-xs-7": "MuiGrid-grid-xs-7-39",
            "grid-xs-8": "MuiGrid-grid-xs-8-40",
            "grid-xs-9": "MuiGrid-grid-xs-9-41",
            "grid-xs-auto": "MuiGrid-grid-xs-auto-31",
            "grid-xs-true": "MuiGrid-grid-xs-true-32",
            "item": "MuiGrid-item-5",
            "justify-xs-center": "MuiGrid-justify-xs-center-21",
            "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-22",
            "justify-xs-space-around": "MuiGrid-justify-xs-space-around-24",
            "justify-xs-space-between": "MuiGrid-justify-xs-space-between-23",
            "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-25",
            "spacing-xs-16": "MuiGrid-spacing-xs-16-27",
            "spacing-xs-24": "MuiGrid-spacing-xs-24-28",
            "spacing-xs-32": "MuiGrid-spacing-xs-32-29",
            "spacing-xs-40": "MuiGrid-spacing-xs-40-30",
            "spacing-xs-8": "MuiGrid-spacing-xs-8-26",
            "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-10",
            "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-11",
            "zeroMinWidth": "MuiGrid-zeroMinWidth-6",
          }
        }
        component="div"
        container={true}
        direction="row"
        item={false}
        justify="space-between"
        lg={false}
        md={false}
        sm={false}
        spacing={0}
        wrap="wrap"
        xl={false}
        xs={false}
        zeroMinWidth={false}
      >
        <div
          className="MuiGrid-container-4 MuiGrid-justify-xs-space-between-23 Title-Container-2"
        >
          <WithStyles(Grid)
            item={true}
            md={5}
            xs={12}
          >
            <Grid
              alignContent="stretch"
              alignItems="stretch"
              classes={
                Object {
                  "align-content-xs-center": "MuiGrid-align-content-xs-center-16",
                  "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-18",
                  "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-17",
                  "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-20",
                  "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-19",
                  "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-15",
                  "align-items-xs-center": "MuiGrid-align-items-xs-center-12",
                  "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-14",
                  "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-13",
                  "container": "MuiGrid-container-4",
                  "direction-xs-column": "MuiGrid-direction-xs-column-7",
                  "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-8",
                  "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-9",
                  "grid-lg-1": "MuiGrid-grid-lg-1-75",
                  "grid-lg-10": "MuiGrid-grid-lg-10-84",
                  "grid-lg-11": "MuiGrid-grid-lg-11-85",
                  "grid-lg-12": "MuiGrid-grid-lg-12-86",
                  "grid-lg-2": "MuiGrid-grid-lg-2-76",
                  "grid-lg-3": "MuiGrid-grid-lg-3-77",
                  "grid-lg-4": "MuiGrid-grid-lg-4-78",
                  "grid-lg-5": "MuiGrid-grid-lg-5-79",
                  "grid-lg-6": "MuiGrid-grid-lg-6-80",
                  "grid-lg-7": "MuiGrid-grid-lg-7-81",
                  "grid-lg-8": "MuiGrid-grid-lg-8-82",
                  "grid-lg-9": "MuiGrid-grid-lg-9-83",
                  "grid-lg-auto": "MuiGrid-grid-lg-auto-73",
                  "grid-lg-true": "MuiGrid-grid-lg-true-74",
                  "grid-md-1": "MuiGrid-grid-md-1-61",
                  "grid-md-10": "MuiGrid-grid-md-10-70",
                  "grid-md-11": "MuiGrid-grid-md-11-71",
                  "grid-md-12": "MuiGrid-grid-md-12-72",
                  "grid-md-2": "MuiGrid-grid-md-2-62",
                  "grid-md-3": "MuiGrid-grid-md-3-63",
                  "grid-md-4": "MuiGrid-grid-md-4-64",
                  "grid-md-5": "MuiGrid-grid-md-5-65",
                  "grid-md-6": "MuiGrid-grid-md-6-66",
                  "grid-md-7": "MuiGrid-grid-md-7-67",
                  "grid-md-8": "MuiGrid-grid-md-8-68",
                  "grid-md-9": "MuiGrid-grid-md-9-69",
                  "grid-md-auto": "MuiGrid-grid-md-auto-59",
                  "grid-md-true": "MuiGrid-grid-md-true-60",
                  "grid-sm-1": "MuiGrid-grid-sm-1-47",
                  "grid-sm-10": "MuiGrid-grid-sm-10-56",
                  "grid-sm-11": "MuiGrid-grid-sm-11-57",
                  "grid-sm-12": "MuiGrid-grid-sm-12-58",
                  "grid-sm-2": "MuiGrid-grid-sm-2-48",
                  "grid-sm-3": "MuiGrid-grid-sm-3-49",
                  "grid-sm-4": "MuiGrid-grid-sm-4-50",
                  "grid-sm-5": "MuiGrid-grid-sm-5-51",
                  "grid-sm-6": "MuiGrid-grid-sm-6-52",
                  "grid-sm-7": "MuiGrid-grid-sm-7-53",
                  "grid-sm-8": "MuiGrid-grid-sm-8-54",
                  "grid-sm-9": "MuiGrid-grid-sm-9-55",
                  "grid-sm-auto": "MuiGrid-grid-sm-auto-45",
                  "grid-sm-true": "MuiGrid-grid-sm-true-46",
                  "grid-xl-1": "MuiGrid-grid-xl-1-89",
                  "grid-xl-10": "MuiGrid-grid-xl-10-98",
                  "grid-xl-11": "MuiGrid-grid-xl-11-99",
                  "grid-xl-12": "MuiGrid-grid-xl-12-100",
                  "grid-xl-2": "MuiGrid-grid-xl-2-90",
                  "grid-xl-3": "MuiGrid-grid-xl-3-91",
                  "grid-xl-4": "MuiGrid-grid-xl-4-92",
                  "grid-xl-5": "MuiGrid-grid-xl-5-93",
                  "grid-xl-6": "MuiGrid-grid-xl-6-94",
                  "grid-xl-7": "MuiGrid-grid-xl-7-95",
                  "grid-xl-8": "MuiGrid-grid-xl-8-96",
                  "grid-xl-9": "MuiGrid-grid-xl-9-97",
                  "grid-xl-auto": "MuiGrid-grid-xl-auto-87",
                  "grid-xl-true": "MuiGrid-grid-xl-true-88",
                  "grid-xs-1": "MuiGrid-grid-xs-1-33",
                  "grid-xs-10": "MuiGrid-grid-xs-10-42",
                  "grid-xs-11": "MuiGrid-grid-xs-11-43",
                  "grid-xs-12": "MuiGrid-grid-xs-12-44",
                  "grid-xs-2": "MuiGrid-grid-xs-2-34",
                  "grid-xs-3": "MuiGrid-grid-xs-3-35",
                  "grid-xs-4": "MuiGrid-grid-xs-4-36",
                  "grid-xs-5": "MuiGrid-grid-xs-5-37",
                  "grid-xs-6": "MuiGrid-grid-xs-6-38",
                  "grid-xs-7": "MuiGrid-grid-xs-7-39",
                  "grid-xs-8": "MuiGrid-grid-xs-8-40",
                  "grid-xs-9": "MuiGrid-grid-xs-9-41",
                  "grid-xs-auto": "MuiGrid-grid-xs-auto-31",
                  "grid-xs-true": "MuiGrid-grid-xs-true-32",
                  "item": "MuiGrid-item-5",
                  "justify-xs-center": "MuiGrid-justify-xs-center-21",
                  "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-22",
                  "justify-xs-space-around": "MuiGrid-justify-xs-space-around-24",
                  "justify-xs-space-between": "MuiGrid-justify-xs-space-between-23",
                  "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-25",
                  "spacing-xs-16": "MuiGrid-spacing-xs-16-27",
                  "spacing-xs-24": "MuiGrid-spacing-xs-24-28",
                  "spacing-xs-32": "MuiGrid-spacing-xs-32-29",
                  "spacing-xs-40": "MuiGrid-spacing-xs-40-30",
                  "spacing-xs-8": "MuiGrid-spacing-xs-8-26",
                  "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-10",
                  "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-11",
                  "zeroMinWidth": "MuiGrid-zeroMinWidth-6",
                }
              }
              component="div"
              container={false}
              direction="row"
              item={true}
              justify="flex-start"
              lg={false}
              md={5}
              sm={false}
              spacing={0}
              wrap="wrap"
              xl={false}
              xs={12}
              zeroMinWidth={false}
            >
              <div
                className="MuiGrid-item-5 MuiGrid-grid-xs-12-44 MuiGrid-grid-md-5-65"
              >
                <h1
                  className="Title-Title-3"
                />
              </div>
            </Grid>
          </WithStyles(Grid)>
          <WithStyles(Grid)
            item={true}
            md={7}
            xs={12}
          >
            <Grid
              alignContent="stretch"
              alignItems="stretch"
              classes={
                Object {
                  "align-content-xs-center": "MuiGrid-align-content-xs-center-16",
                  "align-content-xs-flex-end": "MuiGrid-align-content-xs-flex-end-18",
                  "align-content-xs-flex-start": "MuiGrid-align-content-xs-flex-start-17",
                  "align-content-xs-space-around": "MuiGrid-align-content-xs-space-around-20",
                  "align-content-xs-space-between": "MuiGrid-align-content-xs-space-between-19",
                  "align-items-xs-baseline": "MuiGrid-align-items-xs-baseline-15",
                  "align-items-xs-center": "MuiGrid-align-items-xs-center-12",
                  "align-items-xs-flex-end": "MuiGrid-align-items-xs-flex-end-14",
                  "align-items-xs-flex-start": "MuiGrid-align-items-xs-flex-start-13",
                  "container": "MuiGrid-container-4",
                  "direction-xs-column": "MuiGrid-direction-xs-column-7",
                  "direction-xs-column-reverse": "MuiGrid-direction-xs-column-reverse-8",
                  "direction-xs-row-reverse": "MuiGrid-direction-xs-row-reverse-9",
                  "grid-lg-1": "MuiGrid-grid-lg-1-75",
                  "grid-lg-10": "MuiGrid-grid-lg-10-84",
                  "grid-lg-11": "MuiGrid-grid-lg-11-85",
                  "grid-lg-12": "MuiGrid-grid-lg-12-86",
                  "grid-lg-2": "MuiGrid-grid-lg-2-76",
                  "grid-lg-3": "MuiGrid-grid-lg-3-77",
                  "grid-lg-4": "MuiGrid-grid-lg-4-78",
                  "grid-lg-5": "MuiGrid-grid-lg-5-79",
                  "grid-lg-6": "MuiGrid-grid-lg-6-80",
                  "grid-lg-7": "MuiGrid-grid-lg-7-81",
                  "grid-lg-8": "MuiGrid-grid-lg-8-82",
                  "grid-lg-9": "MuiGrid-grid-lg-9-83",
                  "grid-lg-auto": "MuiGrid-grid-lg-auto-73",
                  "grid-lg-true": "MuiGrid-grid-lg-true-74",
                  "grid-md-1": "MuiGrid-grid-md-1-61",
                  "grid-md-10": "MuiGrid-grid-md-10-70",
                  "grid-md-11": "MuiGrid-grid-md-11-71",
                  "grid-md-12": "MuiGrid-grid-md-12-72",
                  "grid-md-2": "MuiGrid-grid-md-2-62",
                  "grid-md-3": "MuiGrid-grid-md-3-63",
                  "grid-md-4": "MuiGrid-grid-md-4-64",
                  "grid-md-5": "MuiGrid-grid-md-5-65",
                  "grid-md-6": "MuiGrid-grid-md-6-66",
                  "grid-md-7": "MuiGrid-grid-md-7-67",
                  "grid-md-8": "MuiGrid-grid-md-8-68",
                  "grid-md-9": "MuiGrid-grid-md-9-69",
                  "grid-md-auto": "MuiGrid-grid-md-auto-59",
                  "grid-md-true": "MuiGrid-grid-md-true-60",
                  "grid-sm-1": "MuiGrid-grid-sm-1-47",
                  "grid-sm-10": "MuiGrid-grid-sm-10-56",
                  "grid-sm-11": "MuiGrid-grid-sm-11-57",
                  "grid-sm-12": "MuiGrid-grid-sm-12-58",
                  "grid-sm-2": "MuiGrid-grid-sm-2-48",
                  "grid-sm-3": "MuiGrid-grid-sm-3-49",
                  "grid-sm-4": "MuiGrid-grid-sm-4-50",
                  "grid-sm-5": "MuiGrid-grid-sm-5-51",
                  "grid-sm-6": "MuiGrid-grid-sm-6-52",
                  "grid-sm-7": "MuiGrid-grid-sm-7-53",
                  "grid-sm-8": "MuiGrid-grid-sm-8-54",
                  "grid-sm-9": "MuiGrid-grid-sm-9-55",
                  "grid-sm-auto": "MuiGrid-grid-sm-auto-45",
                  "grid-sm-true": "MuiGrid-grid-sm-true-46",
                  "grid-xl-1": "MuiGrid-grid-xl-1-89",
                  "grid-xl-10": "MuiGrid-grid-xl-10-98",
                  "grid-xl-11": "MuiGrid-grid-xl-11-99",
                  "grid-xl-12": "MuiGrid-grid-xl-12-100",
                  "grid-xl-2": "MuiGrid-grid-xl-2-90",
                  "grid-xl-3": "MuiGrid-grid-xl-3-91",
                  "grid-xl-4": "MuiGrid-grid-xl-4-92",
                  "grid-xl-5": "MuiGrid-grid-xl-5-93",
                  "grid-xl-6": "MuiGrid-grid-xl-6-94",
                  "grid-xl-7": "MuiGrid-grid-xl-7-95",
                  "grid-xl-8": "MuiGrid-grid-xl-8-96",
                  "grid-xl-9": "MuiGrid-grid-xl-9-97",
                  "grid-xl-auto": "MuiGrid-grid-xl-auto-87",
                  "grid-xl-true": "MuiGrid-grid-xl-true-88",
                  "grid-xs-1": "MuiGrid-grid-xs-1-33",
                  "grid-xs-10": "MuiGrid-grid-xs-10-42",
                  "grid-xs-11": "MuiGrid-grid-xs-11-43",
                  "grid-xs-12": "MuiGrid-grid-xs-12-44",
                  "grid-xs-2": "MuiGrid-grid-xs-2-34",
                  "grid-xs-3": "MuiGrid-grid-xs-3-35",
                  "grid-xs-4": "MuiGrid-grid-xs-4-36",
                  "grid-xs-5": "MuiGrid-grid-xs-5-37",
                  "grid-xs-6": "MuiGrid-grid-xs-6-38",
                  "grid-xs-7": "MuiGrid-grid-xs-7-39",
                  "grid-xs-8": "MuiGrid-grid-xs-8-40",
                  "grid-xs-9": "MuiGrid-grid-xs-9-41",
                  "grid-xs-auto": "MuiGrid-grid-xs-auto-31",
                  "grid-xs-true": "MuiGrid-grid-xs-true-32",
                  "item": "MuiGrid-item-5",
                  "justify-xs-center": "MuiGrid-justify-xs-center-21",
                  "justify-xs-flex-end": "MuiGrid-justify-xs-flex-end-22",
                  "justify-xs-space-around": "MuiGrid-justify-xs-space-around-24",
                  "justify-xs-space-between": "MuiGrid-justify-xs-space-between-23",
                  "justify-xs-space-evenly": "MuiGrid-justify-xs-space-evenly-25",
                  "spacing-xs-16": "MuiGrid-spacing-xs-16-27",
                  "spacing-xs-24": "MuiGrid-spacing-xs-24-28",
                  "spacing-xs-32": "MuiGrid-spacing-xs-32-29",
                  "spacing-xs-40": "MuiGrid-spacing-xs-40-30",
                  "spacing-xs-8": "MuiGrid-spacing-xs-8-26",
                  "wrap-xs-nowrap": "MuiGrid-wrap-xs-nowrap-10",
                  "wrap-xs-wrap-reverse": "MuiGrid-wrap-xs-wrap-reverse-11",
                  "zeroMinWidth": "MuiGrid-zeroMinWidth-6",
                }
              }
              component="div"
              container={false}
              direction="row"
              item={true}
              justify="flex-start"
              lg={false}
              md={7}
              sm={false}
              spacing={0}
              wrap="wrap"
              xl={false}
              xs={12}
              zeroMinWidth={false}
            >
              <div
                className="MuiGrid-item-5 MuiGrid-grid-xs-12-44 MuiGrid-grid-md-7-67"
              >
                <WithStyles(Buttons)>
                  <Buttons
                    classes={
                      Object {
                        "Buttons": "Buttons-Buttons-101",
                      }
                    }
                  >
                    <div
                      className="Buttons-Buttons-101"
                    >
                      <button
                        className="btn btn__submit margin-0"
                        type="button"
                      >
                        Save Changes
                      </button>
                    </div>
                  </Buttons>
                </WithStyles(Buttons)>
              </div>
            </Grid>
          </WithStyles(Grid)>
        </div>
      </Grid>
    </WithStyles(Grid)>
  </Title>
</WithStyles(Title)>
`;
